load-generator-source/src/main/resources/config/spring-module.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:int="http://www.springframework.org/schema/integration"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+		http://www.springframework.org/schema/beans/spring-beans.xsd
+		http://www.springframework.org/schema/integration
+		http://www.springframework.org/schema/integration/spring-integration.xsd">
+
+    <bean class="org.springframework.xd.loadgenerator.LoadGenerator">
+        <constructor-arg name="messageCount" value="${messageCount}"/>
+        <constructor-arg name="producers" value="${producers}"/>
+        <constructor-arg name="messageSize" value="${messageSize}"/>
+        <property name="autoStartup" value="false"/>
+        <property name="outputChannel" ref="output"/>
+    </bean>
+
+    <int:channel id="output">
+        <int:dispatcher task-executor="taskExecutor"/>
+    </int:channel>
+
+    <bean id="taskExecutor"
```

Comment:
Do you mean the taskExecutor that was selected or the fact that taskExecutor was used at all.